src/test/java/marquez/client/MarquezHttpTest.java

```diff
@@ -143,6 +154,29 @@ public void testPost() throws Exception {
     assertThat(actual).isEqualTo(json);
   }
 
+  @Test
+  public void testPostWithApiKey() throws Exception {
```

Comment:
is the goal to check that the header is set?

Reply:
Yeah, but there's no straightforward way to test the header was set. I'll remove the tests for now.